web/src/components/tasks/business-formation/contacts/ContactsStep.tsx

```diff
@@ -85,9 +85,9 @@ export const ContactsStep = (): ReactElement => {
             defaultAddress={
               "limited-partnership" === state.formationFormData.legalType
                 ? {
-                    addressCity:
-                      state.formationFormData.addressMunicipality?.name ??
-                      state.formationFormData.addressCity,
+                    foreignAddressCity:
```

Comment:
address modal shouldn't change the field name (but you still need the changes right below here)